packages/api-derive/src/staking/info.ts

```diff
@@ -34,13 +43,66 @@ function withStashController (api: ApiInterface$Rx, accountId: AccountId, contro
       stakingLedger: stakingLedger.isSome
         ? stakingLedger.unwrap()
         : undefined,
+      redeemable: redeemableSum(stakingLedger.unwrap().unlocking, eraLength, bestNumber),
       stashId,
       validatorPrefs
     })),
     drr()
   );
 }
 
+function calculateUnlocking (unlockings: Vector<UnlockChunk>, eraLength = new BN(0), bestNumber= new BlockNumber(0)) {
+  // select the Unlockchunks that can't be redeemed yet.
+  const unlockingChunks = unlockings.filter((chunk) => remainingBlocks(chunk.era, eraLength, bestNumber).gtn(0));
+
+  if (unlockingChunks.length) {
+    // group the Unlockchunks that have the same era and sum their values
+    const groupedResult = groupByEra(unlockingChunks);
+    const results: {value: BN, remainingBlocks: BN}[] = [];
```

Comment:
This form `{value: BN, remainingBlocks: BN}` gets used a lot, define it and export from `./types.ts`, import in this file

Reply:
... Additionally, follow the alphabetical suggestions for this type & `Array<MyCoolType>` as opposed to the `[]` shorthand. (Just consistency)